### PR TITLE
Adds the desktop version of our project

### PR DESCRIPTION
**Pull request by mkaya13**
Adds desktop version styling to the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,151 +11,188 @@
...
+                                <a class="navbar-link" href="#portfolio">Portfolio</a>
+                            </li>
+                            <li class="navbar-items">
+                                <a class="navbar-link" href="#about">About</a>
+                            </li>
+                            <li class="navbar-items">
+                                <a class="navbar-link" href="#contact">Contact</a>
+                            </li>
+                        </ul>
                     </nav>
```
Comment (Olamarx) [inline index.html:37]: - [ ] Kindly make your design look like the Figma design starting from the minimum width
## Your Design
image
## Figma Design
image

File: index.html [hunk 2]
```diff
@@ -11,151 +11,188 @@
...
-                    <li class ="title-element">CANOPY</li> 
-                    &nbsp;
-                    <li class ="list-element">
-                        <img src="img/Counter.png">
+        <div class="grid-item flex-container" id="portfolio">
+            <div class ="flex-item">
+                <img class="project-content-1 left-side-picture" src="img/pro-1.svg">
+            
+              
+                <div class="right-side-content">
```
Comment (Olamarx) [inline index.html:81]: - [ ] Kindly make the design responsive from the minimum width. You can do this with the use of min-width
## Your Design
image
## Figma Design
image

File: index.html [hunk 3]
```diff
@@ -164,96 +201,150 @@ <h1 class="title">
...
+                    <p class ="about-paragraph">
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, don’t hestiate to contact me.   
+                    </p> 
+                
+                
+                    <h2 class="about-connect">
+                        LET’S CONNECT
+                    </h2>
+
```
Comment (Olamarx) [inline index.html:268]: - [ ] Kindly make the minimum desktop view to look like the figma. You can get this done with the use of min-width.
## Your design
image
## The Figm
image
a

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @mkaya13 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



- [ ] You did a beautiful job but you just need to make them responsive from the minimum view of the desktop. I mean all the content should show from 768px width upward. 😉 

## Highlights
- Good commit messages ✔️
- Descriptive readme 👍🏼 
- Good use of Git and GitHub flow ✅ 
- All linters passed ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (mkaya13) [reply index.html:268]: Hi, I did not understand what you meant, could you please tell me more comprehensively?

Thank you

Comment (mkaya13) [reply index.html:268]: @Olamarx I did not understand the feedback can you please tell me more comprehensively?

Thank you

**Review by jaflih — APPROVED** [review]
Hi @mkaya13  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
Linters working properly ✔️
Commit messages are meaningful ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**